### PR TITLE
feat(golang): add golang image

### PR DESCRIPTION
**Pull request by plouc**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Versions
 ========
 
+2017.01.24
+----------
+
+* Add a new Golang image
```
Comment (rande) [inline CHANGELOG.md]: Add this note to the ``2017-02-DEV`` futur tag ;)

**Comment by plouc** [comment]
@rande, done

File: CHANGELOG.md (hunk 1 above)
Comment (plouc) [inline CHANGELOG.md]: The tags should be normalized in the Changelog file, there is a mix of `.` & `-`, should be identical to git tags (`.`).

**Comment by plouc** [comment]
The failed build on java 8 seems to have absolutely nothing to do with the changes I introduced…

**Pull request merged**